lib/relations/manyToMany/ManyToManyRelation.js

```diff
@@ -52,7 +52,7 @@ class ManyToManyRelation extends Relation {
     ctx = finalizeJoinModelClass(ctx);
 
     this.joinTableExtras = ctx.joinTableExtras;
-    this.joinTableModelClass = ctx.joinModelClass;
+    this.joinTableModelClass = ctx.joinTableModelClass;
```

Comment:
@koskimas I renamed `ctx.joinModelClass` to `ctx.joinTableModelClass` so that is symmetric with the fields on `Relation`, and also so that it's easier to find actual uses of `Relation.joinModelClass`